.github/workflows/build-ubuntu16.yml

```diff
@@ -0,0 +1,91 @@
+name: Ubuntu 16 Build
+
+on:
+  push:
+    branches: [ main ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+
+      - name: Lowercase repo owner
+        id: repo_owner
+        run: echo "::set-output name=lowercase::$(echo ${{ github.repository_owner }} | tr \"[:upper:]\" \"[:lower:]\")"
+        shell: bash
+
+      - name: Meta for base
+        id: meta_base
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ steps.repo_owner.outputs.lowercase }}/indy-node-container/base
+          tags: |
+            type=raw,value=ubuntu
+
+      - name: Meta for indy_node
+        id: meta_indy_node
+        uses: docker/metadata-action@v3
+        with:
+          images: ghcr.io/${{ steps.repo_owner.outputs.lowercase }}/indy-node-container/indy_node
+          # Note: no latest tag for the ubuntu build
+          flavor: |
+            suffix=-ubuntu
```

Comment:
ubuntu16
and similar change to ubuntu18 in that build action

Reply:
done